client/stripe-link/index.js

```diff
@@ -0,0 +1,84 @@
+const showLinkButton = ( linkAutofill ) => {
+	// Display StripeLink button if email field is prefilled.
+	if ( jQuery( '#billing_email' ).val() !== '' ) {
+		const linkButtonTop =
+			jQuery( '#billing_email' ).position().top +
+			( jQuery( '#billing_email' ).outerHeight() - 40 ) / 2;
+		jQuery( '.wcpay-stripelink-modal-trigger' ).show();
+		jQuery( '.wcpay-stripelink-modal-trigger' ).css(
+			'top',
+			linkButtonTop + 'px'
+		);
+	}
+
+	// Handle StripeLink button click.
+	jQuery( '.wcpay-stripelink-modal-trigger' ).on( 'click', ( event ) => {
+		event.preventDefault();
+		// Trigger modal.
+		linkAutofill.launch( { email: jQuery( '#billing_email' ).val() } );
+	} );
+};
+
+const enableStripeLinkPaymentMethod = ( options ) => {
+	if ( ! document.getElementById( options.emailId ) ) {
+		return;
+	}
+	const api = options.api;
+	const linkAutofill = api.getStripe().linkAutofillModal( options.elements );
+
+	document
+		.getElementById( options.emailId )
+		.addEventListener( 'keyup', ( event ) => {
+			linkAutofill.launch( { email: event.target.value } );
+		} );
+
+	const showButton = options.show_button
+		? options.show_button
+		: showLinkButton;
+	showButton( linkAutofill );
+
+	linkAutofill.on( 'autofill', ( event ) => {
+		const { billingAddress, shippingAddress } = event.value;
+		const fillWith = options.fill_field_method
+			? options.fill_field_method
+			: ( address, nodeId, key ) => {
+					document.getElementById( nodeId ).value =
+						address.address[ key ];
+			  };
+
+		if ( options.complete_shipping ) {
+			fillWith( shippingAddress, options.shipping_fields.line1, 'line1' );
```

Comment:
This line is causing an error in the browser console for me:

```
Uncaught TypeError: Cannot set properties of null (setting 'value')
    at fillWith (index.js:45:6)
    at index.js:67:4
    at ?ver=3.0:1:132443
    at ?ver=3.0:1:311538
    at Array.forEach (<anonymous>)
    at ?ver=3.0:1:311517
    at ?ver=3.0:1:134136
    at Array.forEach (<anonymous>)
    at n._emit (?ver=3.0:1:134019)
    at e._handleMessage (?ver=3.0:1:252218)
```
Steps I did to reproduce:

1. Start with a fresh user (logout, clear cookies or use incognito window)
2. Place an order and opt-in for StripeLink
3. Clear cookies
4. Place a new order using the same email you just enrolled in StripeLink
5. Enter the SripeLink security pin and click on 'Autofill checkout'
6. Observe the issue: error thrown to the browser console and the form is not populated

I recorded this [video](https://d.pr/v/HMr3n8) of the issue in case it´s useful.

I set a breakpoint in this line and the problem seems to be that there is no element in the DOM with the "shipping_address_1" id. 
Maybe I have something wrong in my local setup because as you can check in the video I don't see the `Ship to a different address?` block in the form (the one in the screenshot below)

<img width="476" alt="Screen Shot 2022-07-14 at 11 09 18" src="https://user-images.githubusercontent.com/1553182/178946966-a8f30fee-9d3d-49fb-a061-75502ee0b4f5.png">

Reply:
Good catch. Ineed could be some configuration that hides the shipping address block. I added a check, to complete the field only if that field exists. Thank you!